hw3/hw3_task1.py

```diff
@@ -0,0 +1,16 @@
+'''
+you have a list of elements [1, 2, 3, 4, 5, 6, 7, 8]. Loop through the list using the foreach loop.
+The element with an odd index is put into a new list of tuples where the first element is the index
+and the second is the value. [(index, value)]. accordingly,
+elements with an even index are placed in another list of tuples with the same format as in the case with odd indices.
+'''
+elements = [1, 2, 3, 4, 5, 6, 7, 8]
+list_odd_indices = []
+list_even_indices = []
+for element in elements:
+    if elements.index(element) % 2 == 0:
```

Comment:
you code should work with different sets of data. 
[1, 2, 3, 4, 5, 6, 7, 8] is just an example. 
what if I try it with [1, 2, 2, 2, 4, 4, 4, 1]?